fleetctl/fleetctl.go

```diff
@@ -732,6 +803,64 @@ func lazyCreateUnits(args []string) error {
 	return nil
 }
 
+// matchUnitFiles compares two unitFiles
+// Returns true if the units match, false otherwise.
+func matchUnitFiles(a *unit.UnitFile, b *unit.UnitFile) bool {
+	if a.Hash() == b.Hash() {
+		return true
+	}
+
+	return false
+}
```

Comment:
Yeah, I'm aware of that. My point was that the code in question is trivial, and wrapping it in a function doesn't really simplify the invocation...

(BTW, the function is way too complicated -- just `return a.Hash() == b.Hash()` would do exactly the same... And it would make it more obvious that the function is trivial.)